pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -30,42 +37,17 @@ func NewPersistentVolumeClaimForCR(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Persi
 	}
 }
 
-func NewCoreConfigMap(cr *rhjmcv1alpha1.ContainerJFR) *corev1.ConfigMap {
-	return NewConfigMap(cr.Name, cr.Name, cr.Namespace, map[string]string{"expose.config.fabric8.io/host-key": "CONTAINER_JFR_WEB_HOST"})
-}
-
-func NewCommandChannelConfigMap(cr *rhjmcv1alpha1.ContainerJFR) *corev1.ConfigMap {
-	return NewConfigMap(cr.Name+"-command", cr.Name, cr.Namespace, map[string]string{"expose.config.fabric8.io/host-key": "CONTAINER_JFR_LISTEN_HOST"})
-}
-
-func NewJfrDatasourceConfigMap(cr *rhjmcv1alpha1.ContainerJFR) *corev1.ConfigMap {
-	return NewConfigMap(cr.Name+"-jfr-datasource", cr.Name, cr.Namespace, map[string]string{"expose.config.fabric8.io/url-key": "GRAFANA_DATASOURCE_URL"})
-}
-
-func NewGrafanaConfigMap(cr *rhjmcv1alpha1.ContainerJFR) *corev1.ConfigMap {
-	return NewConfigMap(cr.Name+"-grafana", cr.Name, cr.Namespace, map[string]string{"expose.config.fabric8.io/url-key": "GRAFANA_DASHBOARD_URL"})
-}
-
-func NewConfigMap(name string, appName string, namespace string, annotations map[string]string) *corev1.ConfigMap {
-	return &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:        name,
-			Namespace:   namespace,
-			Annotations: annotations,
-			Labels: map[string]string{
-				"app": appName,
-			},
-		},
-	}
-}
-
-func NewPodForCR(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Pod {
+func NewPodForCR(cr *rhjmcv1alpha1.ContainerJFR, specs *ServiceSpecs) *corev1.Pod {
 	return &corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      cr.Name + "-pod",
 			Namespace: cr.Namespace,
 			Labels: map[string]string{
-				"app": cr.Name,
+				"app":  cr.Name,
+				"kind": "containerjfr",
+			},
+			Annotations: map[string]string{
+				"redhat.com/containerJfrUrl": specs.CoreAddress,
```

Comment:
Just curious, what do you have in mind for this new label and annotation?

Reply:
They're not particularly important, but I added them just so that the pod and the general URL for the frontend can be easily selected and discovered. Basically, the `oc get` line that I modified in the README makes use of them.